ledger-core/v2/virtual/virtual.go

```diff
@@ -72,10 +72,15 @@ func (lr *Dispatcher) Init(ctx context.Context) error {
 		SlotAliasRegistry: &conveyor.GlobalAliases{},
 	}
 
+	machineConfig := conveyorConfig
```

Comment:
why do you need 
```
conveyorConfig := ...
machineConfig := conveyorConfig
```

Reply:
to avoid mixing up logs from normal/application SMs and from SMs of the conveyor. SMs of the conveyor only create noise

Reply:
My question was - why do we need 2 variables, why not to create machineConfig like it was before?

Reply:
oh, I see, sorry it is passed to PulseConveyorConfig, my bad